data/sh12a/results/plan2/NB_Z_narrow_LET__p2.dat

```diff
@@ -1,191 +1,191 @@
-5.99562 6.90266 0.000901296
-6.09437 6.93531 0.000909744
-6.19312 6.96423 0.000870728
-6.29187 6.99743 0.000920505
-6.39062 7.03113 0.000965619
-6.48937 7.06482 0.000860957
-6.58812 7.09968 0.000917864
-6.68687 7.13471 0.000943594
-6.78562 7.16916 0.000839429
-6.88437 7.20482 0.000843005
-6.98312 7.24216 0.000939534
-7.08187 7.2788 0.000910077
-7.18062 7.31792 0.00082941
-7.27937 7.35798 0.000839457
-7.37812 7.39484 0.000923714
-7.47687 7.43587 0.000931008
-7.57562 7.47643 0.000960479
-7.67437 7.5178 0.000950115
-7.77312 7.5613 0.000883041
-7.87187 7.60365 0.000849596
-7.97062 7.64866 0.000930245
-8.06937 7.69179 0.000947786
-8.16812 7.74243 0.000950469
-8.26687 7.78809 0.000923731
-8.36562 7.83567 0.000884465
-8.46437 7.88526 0.00083854
-8.56312 7.9344 0.000961335
-8.66187 7.98611 0.000864807
-8.76062 8.03831 0.000968797
-8.85937 8.09428 0.00100364
-8.95812 8.14965 0.000908415
-9.05687 8.20521 0.000948624
-9.15562 8.26282 0.000871934
-9.25437 8.32417 0.00104558
-9.35312 8.38696 0.000904062
-9.45187 8.44724 0.000910834
-9.55062 8.50962 0.000901714
-9.64937 8.57499 0.000938298
-9.74812 8.64333 0.000900109
-9.84687 8.71398 0.00111365
-9.94562 8.78565 0.00100134
-10.0444 8.85947 0.00092634
-10.1431 8.93641 0.000963551
-10.2419 9.01206 0.0009621
-10.3406 9.09512 0.000986214
-10.4394 9.17823 0.000964639
-10.5381 9.26387 0.00105871
-10.6369 9.35028 0.000935373
-10.7356 9.44321 0.00104962
-10.8344 9.53905 0.000963764
-10.9331 9.63612 0.000930408
-11.0319 9.73835 0.000925855
-11.1306 9.84389 0.000943874
-11.2294 9.95131 0.000898478
-11.3281 10.0672 0.000904101
-11.4269 10.1858 0.0010033
-11.5256 10.3076 0.00100694
-11.6244 10.4357 0.00094931
-11.7231 10.5681 0.000912937
-11.8219 10.7086 0.000942124
-11.9206 10.8549 0.0010223
-12.0194 11.0071 0.000950051
-12.1181 11.1707 0.000990385
-12.2169 11.3418 0.00103527
-12.3156 11.5225 0.00105567
-12.4144 11.7096 0.000989219
-12.5131 11.9137 0.00104043
-12.6119 12.1311 0.00103456
-12.7106 12.3601 0.00119727
-12.8094 12.606 0.00131507
-12.9081 12.8737 0.00125682
-13.0069 13.1682 0.00161165
-13.1056 13.477 0.00165946
-13.2044 13.8229 0.0018805
-13.3031 14.2049 0.00193175
-13.4019 14.6335 0.00240333
-13.5006 15.1079 0.00285734
-13.5994 15.6563 0.00315714
-13.6981 16.2769 0.00363913
-13.7969 17.0062 0.00401723
-13.8956 17.8694 0.00443253
-13.9944 18.8918 0.00553215
-14.0931 20.1309 0.00583271
-14.1919 21.7112 0.00764945
-14.2906 23.753 0.00839551
-14.3894 26.5779 0.00996722
-14.4881 30.6725 0.0111557
-14.5869 36.6942 0.0135064
-14.6856 45.0034 0.0163688
-14.7844 55.5689 0.020023
-14.8831 67.5944 0.020116
-14.9819 80.2745 0.0204892
-15.0806 92.8157 0.024785
-15.1794 104.832 0.0276756
-15.2781 115.972 0.0330363
-15.3769 126.333 0.0456349
-15.4756 135.918 0.0669189
-15.5744 144.78 0.0998813
-15.6731 152.886 0.159295
-15.7719 159.776 0.316835
-15.8706 167.534 0.578314
-15.9694 175.267 1.47394
-16.0681 170.647 6.68979
-16.1669 56.3363 7.16404
-16.2656 9.75748 4.44014
+-2.20063 5.2816 3.76115e-06
```

Comment:
Interesting, a slight change in LET for primary protons. 1%

Reply:
this makes sense. In previous version of SH12A heavy ions (with gen=0 which are target nucleus put into motion elastically) where wrongly scored as protons, which inflated LET a bit.

Reply:
I agree !